client/src/components/DebouncedTextInput/DebouncedTextInput.stories.js

```diff
@@ -0,0 +1,17 @@
+import _ from "lodash";
+import React from "react";
+
+import { DebouncedTextInput } from "./DebouncedTextInput.js";
+
+export default {
+  title: "Input/DebouncedTextInput",
+  component: DebouncedTextInput,
+};
+
+// not sure if the component works or not
+const Template = (args) => <DebouncedTextInput {...args} />;
```

Comment:
It does, the important thing is the debounced callback. Will need to expand this storybook a bit more to demonstrate that. Let me know if you want hints :+1:

Reply:
Sure!